src/views/Diagnose/components/WebSocketItem.vue

```diff
@@ -48,6 +48,34 @@
               <el-input v-model.number="connection.keepalive" placeholder="60" />
             </el-form-item>
           </el-col>
+          <el-col :span="8">
+            <el-form-item
+              prop="clean"
+              :label="
+                connection.protocolversion === MQTT_V5_VALUE ? 'Clean Start' : 'Clean Session'
+              "
+            >
+              <BooleanSelect v-model="connection.clean" />
+            </el-form-item>
+          </el-col>
+          <el-col :span="8" v-if="connection.protocolversion === MQTT_V5_VALUE">
+            <el-form-item prop="sessionExpiryInterval" :label="$t('Tools.sessionExpiryInterval')">
+              <el-input
+                class="input-with-unit"
+                v-model.number="connection.sessionExpiryInterval"
+                :placeholder="$t('Tools.neverExpire')"
```

Comment:
Please refer to https://github.com/emqx/MQTTX/blob/main/src/utils/mqttUtils.ts#L13

The never expire value is not default.